argilla/src/argilla/settings/_io/_hub.py

```diff
@@ -261,8 +265,25 @@ def _define_settings_from_features(
     return settings
 
 
-def build_settings_from_repo_id(repo_id: str, feature_mapping: Optional[Dict[str, str]] = None) -> "Settings":
-    dataset_features = _get_dataset_features(repo_id)
+def build_settings_from_repo_id(
+    repo_id: str,
+    feature_mapping: Optional[Dict[str, str]] = None,
+    config: Optional[str] = None,
+    subset: Optional[str] = None,
```

Comment:
why not using just one of them? 
```suggestion
    subset: Optional[str] = None,
```